components/TestSubmissionsLink/TestSubmissionsLink.tsx

```diff
@@ -0,0 +1,26 @@
+﻿import React from "react";
+import textStyles from './TestSubmissionsLink.module.scss';
+import Link from "next/link";
+
+interface TestSubmissionsLinkProps {
+    test: string;
+}
+
+export function TestSubmissionsLink(props: TestSubmissionsLinkProps) : JSX.Element {
+    
+    const ref: string = "/test-submissions/"; 
+   
+    //TODO: how will results appear to admin? for now put textarea as placeholder
```

Comment:
I think we'll want them to be able to see the code with some nice syntax highlighting.
Probably worth speaking to @Espe-on who know's lots about this! :)